### PR TITLE
chore: Use `import/order` eslint rule

### PR DESCRIPTION
**Pull request by lorensr**
Roey suggested this automated solution in lieu of running VS Code's `Organize imports` command when working in a file.

Left the default order, but it's customizable if we want: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

ESLint recommends not using VS Code's `Organize imports` command (which works differently): https://github.com/import-js/eslint-plugin-import/issues/1742#issuecomment-968376099

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "wait-namespace": "node ./scripts/wait-on-temporal.mjs",
     "lint": "eslint packages/*/src --ext .ts --no-error-on-unmatched-pattern && prettier --end-of-line auto --check .",
     "lint.prune": "ts-prune --error -p tsconfig.prune.json --ignore \"used in module\" --skip \".d.ts\"",
-    "format": "prettier --write .",
+    "format": "eslint packages/*/src --ext .ts --no-error-on-unmatched-pattern --fix && prettier --write .",
```
Comment (bergundy) [inline package.json:34]: nit: not related to formatting

**Review by bergundy — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lorensr) [reply package.json]: maybe this?

https://github.com/temporalio/sdk-typescript/pull/965/commits/6e02630cb50f97d0f77661327969cc3f91cd91d7

looking for one command to run locally before committing

**Comment by mjameswh** [comment]
@loren Can we try adding `@temporalio/*` as "internal packages", and have them appears after external packages?

I think that would be something like this:

```
# .eslintrc.yml
settings:
  "import/internal-regex": ^@temporalio/
  "import/order": [
    "error",
    {
      "groups": [
        "builtin",
        "external",
        "internal",
        "parent",
        "sibling",
        "index"
      ]
    }
  ]
 ```

**Comment by mjameswh** [comment]
I added definition of '@temporalio/*' as "internal packages".

**Comment by lorensr** [comment]
Thanks! ☺️

**Pull request merged**